lab-bundle-spec.md

```diff
@@ -271,6 +271,23 @@ No additional attributes
   id: primary_domain
 ```
 
+##### GCP Shell (gcp_shell)
+
```

Comment:
I'm still not sure how to best express "Future Functionality" in this spec, given that this repo is public to the world. I propose the following for now.

```suggestion

> _PROPOSED ONLY: GCP Shell is not yet supported by Qwiklabs runtime._

```